region/brinstar/kraid/Baby Kraid Room.json

```
@@ -163,6 +163,44 @@
         "again the windows can be more narrow depending on what is required in the next room."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "canCarefulJump",
```

```suggestion
        "canTrickyJump",
```
Killing miniKraid with speed with the pirates alive is trickyJump, to be faster than the miniSpikes